### PR TITLE
Fix spelling in README (bable to babel)

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Brackets-Harmony has serveral [gulp](http://gulpjs.com/) tasks that leverage too
 
 
 #### React JSX
-[bablejs](https://babeljs.io/) natively supports tranpiling [react's jsx](https://facebook.github.io/react/docs/jsx-in-depth.html), so you can easily write your components using [react](http://facebook.github.io/react/) using jsx syntax.
+[babeljs](https://babeljs.io/) natively supports tranpiling [react's jsx](https://facebook.github.io/react/docs/jsx-in-depth.html), so you can easily write your components using [react](http://facebook.github.io/react/) using jsx syntax.
```
Comment (MiguelCastillo) [inline README.md]: Cool thanks!!

Comment (MiguelCastillo) [inline README.md]: There is one more if you want to add it to your PR.  `tranpiling`...

**Comment by Mark-Simulacrum** [comment]
Fixed some more misspellings.

**Comment by MiguelCastillo** [comment]
HA! holy smokes! thanks dude!

**Pull request merged**